docker/azure-services/local.settings.json.template

```diff
@@ -18,6 +18,7 @@
     "AV_API_USER_ID": "",
     "AV_API_BASE_URL": "",
     "AV_API_TOKEN_URL": "",
+    "AV_API_SB_CONNECTION_STRING": "",
     "AzureSignalRConnectionString": "Endpoint=http://azure_services;Port=8888;AccessKey=ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789ABCDEFGH;Version=1.0;",
     "WEBSITE_NODE_DEFAULT_VERSION": "~16"
   },
```

Comment:
Did not notice this before, should these details be committed to github

Reply:
Yes AzureSignalRConnectionString is just the dummy/default connection details for the containerised version of signalr we use for local development/testing